apps/web-client/src/lib/components/Tables/InventoryTables/BookPriceCell.svelte

```diff
@@ -22,5 +22,5 @@
 {:else}
 	<!-- Price only is shown regardless of the row being a book row without discount applied, or a custom item row -->
 	{@const { price } = data}
-	<span data-property="full-price">{price ? `€${price?.toFixed(2)}` : "N/A"} </span>
+	<span data-property="full-price">{price || price === 0 ? `€${price?.toFixed(2)}` : "N/A"} </span>
```

Comment:
No need for fallback here (`N/A`) -- prices should always be defined (fall back to 0) at query level. If, by chance, this fails, the e2e will pick it up (expecting `€0` and not `N/A` -- or in faulty case `€undefined`)